toxav/rtp.c

```diff
@@ -382,7 +382,8 @@ int handle_rtp_packet (Messenger *m, uint32_t friendnumber, const uint8_t *data,
                 session->mp = new_message(ntohs(header->tlen) + sizeof(struct RTPHeader), data, length);
 
                 /* Reposition data if necessary */
-                if (ntohs(header->cpart));
+                if (ntohs(header->cpart))
+                    ;
```

Comment:
this is something that's been in toxav for some time. I had it fixed in a ToxAV branch I was working on, but I abandoned it before it got merged. That's the only reason it still exists, irungentoo and mannol were both aware, but waiting to merge this fix with my AV changes.